packages/alpha/src/__tests__/api/simulatorRoutinesApi.spec.ts

```diff
@@ -82,7 +81,7 @@ describeIf('simulator routines api', () => {
         externalId: modelRevisionExternalId,
         modelExternalId,
         description: 'test sim model revision description',
-        fileId: 3747718694331206,
+        fileId: 6396395402204465,
         metadata: {},
```

Comment:
could you remove this metadata field from the type? we should not use it anymore